plugins/tech-stack/src/config/ToolsDefaultConfig.json

```diff
@@ -0,0 +1,480 @@
+{
+  "config": [
+    {
+      "type": "Gitlab",
+      "src": "assets/img/logos/GitLab_logo.png",
```

Comment:
Are these assets intentionally left out of the PR?

A tricky bit about this plugin is whether we can have these assets in the repo or not. It's likely gonna take a while to figure out so please bear with us 😅 

We could also merge the PR without the config for the tools too, and let that live in the app for now.

Reply:
now we use full uri